src/components/MultiHopTrade/components/TradeAmountInput.tsx

```diff
@@ -176,6 +177,12 @@ export const TradeAmountInput: React.FC<TradeAmountInputProps> = memo(
       [assetSymbol, balance, fiatBalance, isFiat, translate],
     )
 
+    const handleOnMaxClick = useMemo(() => {
+      if (!onMaxClick) return
+
+      return () => onMaxClick(isFiat)
```

Comment:
honors `onMaxClick` being optional, and `isFiat` parameter for `onChange`

Reply:
Disregard the optional - see @kaladinlight 's comment in https://github.com/shapeshift/web/pull/5961/files#r1446451716 and the matching commit c57643304803e4f834f36ba2d3653f02a2a6db39